source/mainnet/net/guides/run-node-windows.rst

```diff
@@ -209,36 +209,6 @@ When nodes are stopped or started, this is also recorded in the system event log
    .. image:: ../images/run-node/Node-setup-win-7.png
          :width: 50%
 
-
-Synchronize a node with the network
-===================================
-
-When you start a node for the first time, it can take a while to synchronize the node with the rest of the network, since it has to get all blocks from its peers.
-
-You can improve the performance by downloading the blocks from an out-of-band catchup service before starting the node. While it will still take time to process the blocks, it will typically be faster than requesting them from peers.
-
-#. Search for *configure concordium node* in the **Search** bar, and then select **Configure Concordium Node Service**. The configuration file opens in **Notepad**.
-
-#. Specify the URL to the block file index in the configuration file:
```

Comment:
Instead of deleting this it would be better to revise it I think.

After the first sentence we'd say something along the lines of "That is why all node distributions since 6.1 come with out of band catchup enabled.  This will speed up the initial catchup and during out of band catchup the node will not have any peers."

The out of band catchup can be kept enabled even after the node is caught up, but is not necessary. If you wish to disable it you can ...

And then show how to disable it below by removing the relevant configuraiton.